temoa/_internal/table_writer.py

```
@@ -81,13 +77,122 @@ def __init__(self, config: TemoaConfig, epsilon: float = 1e-5) -> None:
+        exc_type: type[BaseException] | None,
+        exc_value: BaseException | None,
+        traceback: TracebackType | None,
+    ) -> None:
+        self.close()
+
+    def close(self) -> None:
+        """Explicitly close the database connection."""
+        if self.con:
+            try:
+                self.con.close()
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)